.github/configs/renovate-config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    platform: 'github',
+    gitAuthor: 'renovate[bot] <renovate[bot]@users.noreply.github.com>',
+    autodiscover: false,
+    allowPostUpgradeCommandTemplating: true,
+    allowedPostUpgradeCommands: ["make mockgen"],
+    // Presets from the old renovate.json
```

Comment:
No need for this line, the PR documents the change.

Reply:
resolved.